<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>513</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Kalpurush</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Corpus Development Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 221, 255);


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalpurush</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Kalpurush&quot;;</string>
    </property>
    <property name="text">
     <string>Source Sentence:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalpurush</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Kalpurush&quot;;</string>
    </property>
    <property name="text">
     <string>Translation: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="target">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalpurush</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 20pt &quot;Kalpurush&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="translateButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Submit Translation</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_validate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Kalpurush&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QLabel" name="source">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>33</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalpurush</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(223, 223, 223);color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openButton"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="openButton">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
